# 夹取时的姿态和位置
warehouse/Pose grasp_pos

# 接近时的距离和方向
geometry_msgs/Vector3 grasp_approach

# 撤离时的距离和方向
geometry_msgs/Vector3 grasp_retreat

# 抬高高度
float64 up

# 夹取时夹持器姿态
int16 grasp_posture

# 夹取前夹持器姿态
int16 pre_grasp_posture

